content/Arduino Cloud/General Cloud/Supported-Wi-Fi-bands.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Supported Wi-Fi bands"
+id: 14144656051740
+---
+
+## Use 2.4 GHz Band for IoT Devices
+
+Arduino boards and most IoT devices can only be used with 2.4 GHz band networks. The 2.4 GHz band offers better range and signal strength through obstacles compared to 5 GHz.
+
+## How to Check if Your Network is 2.4 GHz
+
+Follow these steps to identify the frequency band of your Wi-Fi.
+
+**Windows:**
+
+1. Open the settings menu in Windows and select the **Network and Internet** section.
+
+1. Under the wireless network settings, click on the **Hardware properties** link.
+
+1. A properties page will open. Here, you can find details about the wireless network, including the network band (GHz).
+
+**macOS:**
+
+1. Click on the Apple menu in the top-left corner of your screen and select **System Preferences**.
+
+1. In the System Preferences window, click on the **Network** icon.
+
+1. Select the Wi-Fi connection from the list on the left-hand side. Click on the **Advanced** button in the bottom-right corner.
+
+1. In the **Wi-Fi** tab, you'll find information about the Wi-Fi connection, including the frequency band (GHz).
```

Comment:
```suggestion
1. Connect to the Wi-Fi network.

1. Click on the Wi-Fi icon on the right side of the top navigation bar while holding down the Option/Alt key.

1. Look for the entry starting with Channel. This will specify if the network is in the 2.4 GHz or 5 GHz band.

```
Updated instructions for macOS